library/Fission/Types.hs

```diff
@@ -191,3 +198,32 @@ instance MonadRemoteIPFS Fission where
         |> mkClientEnv manager
         |> runClientM query
         |> liftIO
+
+-- 💾 Database Related Instances
+
+instance MonadDB Fission where
+  runDB transaction = do
+    pool <- asks dbPool
+    SQL.runSqlPool transaction pool
+
+instance User.MonadDBQuery Fission where
+  getByUsername = User.getByUsername
+  getHerkouAddonByUserId = User.getHerkouAddonByUserId
+  getHerkouAddonByUUID = User.getHerkouAddonByUUID
+
+instance User.MonadDBMutation Fission where
+  createWithHeroku = User.createWithHeroku
+  create = User.create
+  updatePassword = User.updatePassword
+  destroy = User.destroy
+  destroyHerokuAddon = User.destroyHerokuAddon
+
+instance User.CID.MonadDBQuery Fission where
+  getByUserId = User.CID.getByUserId
+  getByCids = User.CID.getByCids
+
+instance User.CID.MonadDBMutation Fission where
+  create = User.CID.create
+  createX = User.CID.createX
+  destroyAll = User.CID.destroyAll
+  destroyExact = User.CID.destroyExact
```

Comment:
@expede I wasn't able to had `Fission` default to using these base implementations without specifying the instance.

Heres the error I was receiving below.
![image](https://user-images.githubusercontent.com/1325608/72193473-e531aa80-33c6-11ea-9b76-dcc57667de46.png)

Any ideas?